### PR TITLE
feat:add landing page

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
add landing page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <title>Document</title>
```
Comment (junaidshaikh-js) [inline index.html]: Title name should be something meaningful

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  </head>
+  <body>
+    <!-- navbar -->
+
+    <header class="nav flex_space">
+      <div class="left">
```
Comment (junaidshaikh-js) [inline index.html:20]: wrap navigation inside 'nav' tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      <div class="right social_btn">
+        <ul class="flex">
+          <li class="nav_list pointer">
+            <a href="https://github.com/ajay-mandaviya">
+              <i class="fab fa-github-square fa-2x"></i>
+            </a>
+          </li>
+          <li class="nav_list pointer">
+            <a href="https://www.linkedin.com/in/ajay-mandaviya-222900191/">
+              <i class="fab fa-twitter-square fa-2x"></i>
```
Comment (junaidshaikh-js) [inline index.html:44]: add 'title' attribute to your icons.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+          <button class="btn btn-secondary">
+            <a href="./component/avatar/avatar.html">Get Start</a>
+          </button>
+        </div>
+      </div>
+    </main>
+
+    <!-- install link -->
+    <div class="link-container" id="install">
+      <h1 class="heading">Installation</h1>
```
Comment (junaidshaikh-js) [inline index.html:78]: There should be only one 'h1' per page, read about heading hierarchy

File: index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+      <h2>Css style</h2>
+      <span class="description"
```
Comment (junaidshaikh-js) [inline index.html]: Here `p` tag should be more appropriate.

**Review by deepak29-git — COMMENTED** [review]
everything looks good

File: index.html (hunk 1 above)
Comment (ajay-mandaviya) [reply index.html]: oh Really forgot to change

**Pull request merged**